src/test/java/com/gooddata/http/client/GoodDataHttpClientAT.java

```diff
@@ -42,7 +43,7 @@ public class GoodDataHttpClientAT {
     @Test
     public void gdcLogin() throws IOException {
         final HttpClient client = createGoodDataClient(login, password, httpHost);
-
+        assertNotNull(client);
```

Comment:
I think this might be redundant, if there will login or password null, it is already validated in LoginSSTRetrievalStrategy constructor

Reply:
i added it since sonarqube advised that we should assert result in one test case. I don't know which is better to assert then i chose "client"

Reply:
@thaoctthanh Shall we put some suppression annotation or `//NOSONAR` as well? What was the severity level?

Reply:
the severity level is blocker but someone said this is also the limit of sonar rule https://stackoverflow.com/questions/38764125/sonarqube-issue-add-at-least-one-assertion-to-this-test-case-for-unit-test-wit so i think it's safe to put put a suppression annotation